handlers/bmc_itsm7_change_work_info_create_ce_v2/handler/init.rb

```diff
@@ -0,0 +1,512 @@
+  # General handler utility functions
+  ##############################################################################
+
+  # This method is an accessor for the @config[:forms] variable that caches
+  # form definitions.  It checks to see if the specified form has been loaded
+  # if so it returns it otherwise it needs to load the form and add it to the
+  # cache.
+  def get_remedy_form(form_name)
+    if @config[:forms][form_name].nil?
+      @config[:forms][form_name] = ArsModels::Form.find(form_name, :context => @config[:context])
+    end
+    if @config[:forms][form_name].nil?
+      raise "Could not find form " + form_name
+    end
+    @config[:forms][form_name]
+  end
+
+  def preinitialize_on_first_load_remote(input_document, form_names)
+    puts "Preinitializing ARS Models" if @debug_logging_enabled
+
+    remedy_context = ArsModels::Context.new(
+      :server         => @info_values['ars_server'],
+      :username       => @info_values['ars_username'],
+      :password       => @info_values['ars_password'],
+      :port           => @info_values['ars_port'],
+      :prognum        => @info_values['ars_prognum'],
+      :authentication => @info_values['ars_authentication']
+    )
+
+    # Build up a new configuration
+    if @disable_caching
+      @config = {
+        #:properties => properties,
+        :context => remedy_context,
+        :forms => form_names.inject({}) do |hash, form_name|
+          hash.merge!(form_name => ArsModels::Form.find(form_name, :context => remedy_context))
+        end
+      }
+    else
+      @@config = {
+        #:properties => properties,
+        :context => remedy_context,
+        :forms => form_names.inject({}) do |hash, form_name|
+          hash.merge!(form_name => ArsModels::Form.find(form_name, :context => remedy_context))
+        end
+      }
+    end
+
+    puts "Models Initialized" if @debug_logging_enabled
+  end
+  
+  # This is a template method that is used to escape results values (returned in
+  # execute) that would cause the XML to be invalid.  This method is not
+  # necessary if values do not contain character that have special meaning in
+  # XML (&, ", <, and >), however it is a good practice to use it for all return
+  # variable results in case the value could include one of those characters in
+  # the future.  This method can be copied and reused between handlers.
+  def escape(string)
+    # Globally replace characters based on the ESCAPE_CHARACTERS constant
+    string.to_s.gsub(/[&"><]/) { |special| ESCAPE_CHARACTERS[special] } if string
+  end
+  # This is a ruby constant that is used by the escape method
+  ESCAPE_CHARACTERS = {'&'=>'&amp;', '>'=>'&gt;', '<'=>'&lt;', '"' => '&quot;'}
+
+  def handle_results(entryId, error_msg)
+    <<-RESULTS
+    <results>
+      <result name="Entry Id">#{ERB::Util.html_escape(entryId)}</result>
+      <result name="Handler Error Message">#{ERB::Util.html_escape(error_msg)}</result>
```

Comment:
Since you are defining the escape method, just use that instead of ERB::Util.html_escape(). This is more consistent with other handlers.